packages/client/src/views/SysAdmin/Config/Application.tsx

```diff
@@ -236,17 +290,40 @@ class ApplicationConfigComponent extends React.Component<Props, State> {
   constructor(props: Props) {
     super(props)
     this.state = {
-      activeTabId: TabId.GENERAL
+      activeTabId: TabId.GENERAL,
+      changeModalName: EMPTY_STRING,
+      showNotification: false,
+      notificationStatus: NOTIFICATION_TYPE.IN_PROGRESS,
+      notificationMessages: EMPTY_STRING
+    }
+  }
+
+  changeValue = (notificationStatus: NOTIFICATION_TYPE, messages: string) => {
+    if (notificationStatus !== NOTIFICATION_TYPE.ERROR) {
+      this.hideModal()
+      this.setState({
+        showNotification: true,
+        notificationStatus: notificationStatus,
+        notificationMessages: messages
+      })
     }
-    this.changeTab = this.changeTab.bind(this)
   }
 
   changeTab(id: string) {
     this.setState({ activeTabId: id })
   }
 
+  hideModal = () => {
+    this.setState({ changeModalName: '' })
+  }
+
+  showModal = () => {
+    return !!!this.state.changeModalName ? false : true
```

Comment:
This can be simplified in my opinion

Reply:
Changed the implement and kept a single toggle function to show and hide the modal.